### DISCUSSION TITLE
How to change the icon of the android app?

### DISCUSSION THREAD

**Discussion by ccinsz** [beeware/briefcase#1345] [Q&A]
Hi, there are two ways I try to change the app icon, but it doesn't work.
1. def main():
    return demo0529(icon="resources/logo.ico")
Add the path of the icon, but the app raises the error, like
WARNING: Can't find icon resources/demo0706; falling back to default icon

2. replace the icon image of the resources, delete the android directory, and rerun "briefcase create android", but it also show the old icon.

**Comment by freakboy3742** [comment]
You can't change the icon of an Android app at runtime -it's a property of the packaged application, so that's not going to do much. However, if you're getting an error that it "can't find icon", that's because the icon hasn't been packaged correctly. It's impossible to say why it's failing without details.

It's also impossible to say why your new icon isn't showing up, because you haven't provided any details about what you've done that would enable me to reproduce your problem. All I can do is tell you that the feature *does* work, if it's configured correctly. The [Toga Testbed app](https://github.com/beeware/toga/tree/main/testbed) is an example of an app with a custom icon.

**Reply by ccinsz** [reply]
Thanks for your help.
